### PR TITLE
fix: better ignore rules for npm distribution

### PR DESCRIPTION
**Pull request by dwieeb**
fixes https://github.com/ionic-team/capacitor-plugins/issues/27

see https://github.com/ionic-team/create-capacitor-plugin/pull/3

### CODE REVIEW

File: motion/.npmignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+android/.gradle
+ios/Pods
+ios/Podfile.lock
```
Comment (jcesarmobile) [inline motion/.npmignore:3]: motion is web only, so doesn't have any of this

**Comment by jcesarmobile** [comment]
I think it should ignore everything that is in the gitignore too

File: motion/.npmignore (hunk 1 above)
Comment (dwieeb) [reply motion/.npmignore:3]: I considered this, but I'm fine specializing it

**Comment by dwieeb** [comment]
> I think it should ignore everything that is in the gitignore too

Okay, I investigated this. git and npm treat gitignore slightly differently.

- We'll need to remove all trailing slashes in the gitignore rules for our template: https://github.com/ionic-team/create-capacitor-plugin/pull/4 
- Here's the diff after doing that:

```diff
diff --git a/0 b/1
index 70a8c7d..6756aa0 100644
--- a/0
+++ b/1
@@ -1,3 +1,4 @@
+.prettierignore
 CHANGELOG.md
 CapacitorTextZoom.podspec
 LICENSE
@@ -44,3 +45,8 @@ ios/PluginTests/Info.plist
 ios/PluginTests/PluginTests.swift
 ios/Podfile
 package.json
+rollup.config.js
+src/definitions.ts
+src/index.ts
+src/ios.ts
+tsconfig.json
```

So, there are still files we shouldn't distribute (`src/`, configuration files, etc)

**Pull request merged**